dockstore-webservice/src/main/java/io/dockstore/webservice/core/Tool.java

```diff
@@ -315,7 +315,7 @@ public String getToolPath() {
 
 
     @Enumerated(EnumType.STRING)
-    @JsonProperty("registry_provider")
+    @JsonProperty("registry")
```

Comment:
this name to be getting less descriptive actually, what is going on here actually?

Reply:
Old client require the 'registry' json property to be a string.  So changing new webservice to return that string

Reply:
Please add that in the javadoc for future use/removal